src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2770,8 +2770,22 @@ class AccessibilityHelpWidget extends Widget implements ITerminalWidget {
 		}
 		content.push(this._descriptionForCommand(TerminalCommandId.OpenDetectedLink, openDetectedLink, openDetectedLinkNoKb));
 		content.push(readMore, dismiss);
-		const joinedContent = content.join('\n\n');
-		this._contentDomNode.domNode.appendChild(renderFormattedText(joinedContent));
-		this._contentDomNode.domNode.setAttribute('aria-label', joinedContent);
+		for (const co of content) {
+			const elt = renderFormattedText(co);
+			if (co === readMore) {
+				const link: HTMLElement = document.createElement('a');
+				elt.appendChild(link);
+				link.setAttribute('href', 'https://code.visualstudio.com/docs/editor/accessibility#_terminal-accessibility');
+				this._register(dom.addStandardDisposableListener(link, 'keydown', event => {
+					if (event.keyCode === KeyCode.Enter || event.keyCode === KeyCode.Space) {
+						this._openerService.open('https://code.visualstudio.com/docs/editor/accessibility#_terminal-accessibility');
+					}
+				}));
+			}
+			elt.ariaLabel = co;
+			this._contentDomNode.domNode.appendChild(elt);
+		}
+		this._contentDomNode.domNode.setAttribute('aria-label', introMessage);
```

Comment:
I don't think we want all of the content in the aria label since a screen reader user can navigate it.

Reply:
Instead i'm setting the aria label of each element

Reply:
Doesn't the aria-label of a parent override the that of a child? This was one of the problems we were seeing with the new buffer view